pkg/gateway/services/machine.go

```diff
@@ -13,13 +13,21 @@ import (
 
 func (gws *GatewayService) ListMachines(ctx context.Context, in *pb.ListMachinesRequest) (*pb.ListMachinesResponse, error) {
 	authInfo, _ := auth.AuthInfoFromContext(ctx)
+
+	gpus, err := gws.providerRepo.GetGPUAvailability(gws.appConfig.Worker.Pools)
```

Comment:
@luke-lombardi with this, we will only see external provider GPUs, not local ones e.g. AWS. Is this the ideal behavior?

Reply:
yeah that's fine I think